test/README.md

```diff
@@ -0,0 +1,97 @@
+This directory contains integration tests that test bitcoind and its
```

Comment:
Should dashify this file

s/bitcoind/dashd/
s/bitcoin-qt/dash-qt/

Reply:
https://github.com/dashpay/dash/pull/2912/commits/2147d78c1b1271b63b1965a9e8b0f99e1c698dbb